packages/cozy-client/src/associations/HasManyInPlace.js

```diff
@@ -29,6 +29,27 @@ export default class HasManyInPlace extends Association {
     return this.target[this.name]
   }
 
+  addById(id) {
+    const rel = this.getRelationship()
+    rel.push(id)
+  }
+
+  removeById(id) {
+    const rel = this.getRelationship()
+    const index = rel.indexOf(id)
+    rel.splice(index, 1)
```

Comment:
Maybe you want to test the existence first? Splicing on `-1` has [undesirable effects](https://repl.it/@y_lohse/PrizeDentalProcedures).

Reply:
Rookie mistake :/ Thanks, well spotted !

Fixed in https://github.com/cozy/cozy-client/pull/158/commits/52b169de8fdde28c908e3cd54df877525e3ceea9